src/status_im2/contexts/chat/home/contact_request/view.cljs

```diff
@@ -45,8 +45,12 @@
    [rn/view {:style (style/contact-requests-icon)}
     [quo/icon :i/pending-user {:color (colors/theme-colors colors/neutral-50 colors/neutral-40)}]]
    [rn/view {:style {:margin-left 8}}
-    [rn/text {:weight :semi-bold} (i18n/label :t/pending-requests)]
-    [rn/text
+    [quo/text
+     {:size   :paragraph-1
+      :weight :semi-bold
+      :style  {:color (colors/theme-colors colors/black colors/white)}}
```

Comment:
`colors/neutral-100`

Reply:
not sure why colors/black is in colors file, there is no such color in [foundations file](https://www.figma.com/file/v98g9ZiaSHYUdKWrbFg9eM/Foundations?node-id=3056-75&t=zFgYafIsWDWMpQjJ-0) for app theme use case